inference-engine/src/readers/ir_reader_v7/ie_layer_validators.cpp

```diff
@@ -456,24 +456,6 @@ void ScaleShiftValidator::checkShapes(const CNNLayer* layer, const std::vector<S
     checkNumOfInput(inShapes, {1});
 }
 
-void TileValidator::checkParams(const CNNLayer* layer) {
```

Comment:
Why do we need these changes in v7 reader?

Reply:
These changes came from PR 1384. Suppose they were required then.

Reply:
Reverted.